scripts/docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ -z "${SOURCECRED_GITHUB_TOKEN}" ]; then
+    echo "Export SOURCECRED_GITHUB_TOKEN with the container."
+    echo "docker run --env SOURCECRED_GITHUB_TOKEN=$(cat .token) -p 8080:8080 sourcecred <repository>"
```

Comment:
This will actually print out the token in the message; the command
substitution will be expanded. That needs to be escaped, so that the
token doesn’t end up in logs or the user’s terminal window.

Reply:
It can't - the command is run from inside the container, and the .token is in the .dockerignore. But it _will_ spit out an error to not have the file, so I'll fix the quotes for that!

Reply:
Fixed.